### PR TITLE
add config-yaml sample

### PR DESCRIPTION
**Pull request by YarBor**
Add config-yaml sample, Show how to load yaml in new api

### CODE REVIEW

File: config-yaml/go-server/cmd/main.go [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+package main
+
+import (
+	"context"
+	"dubbo.apache.org/dubbo-go/v3"
+	_ "dubbo.apache.org/dubbo-go/v3/imports"
+	"errors"
+	"fmt"
+
+	greet "github.com/apache/dubbo-go-samples/config-yaml/proto"
```
Comment (FinalT) [inline config-yaml/go-server/cmd/main.go:27]: plz format imports

File: integrate_test/config-yaml/tests/integration/context_test.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+package integration
+
+import (
+	"context"
+	"github.com/dubbogo/gost/log/logger"
+	"testing"
+
+	greet "github.com/apache/dubbo-go-samples/config-yaml/proto"
+	"github.com/stretchr/testify/assert"
```
Comment (FinalT) [inline integrate_test/config-yaml/tests/integration/context_test.go:26]: ditto.

File: config-yaml/go-client/cmd/main.go [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+ */
+
+package main
+
+import (
+	"context"
+	"dubbo.apache.org/dubbo-go/v3"
+	_ "dubbo.apache.org/dubbo-go/v3/imports"
+	greet "github.com/apache/dubbo-go-samples/config-yaml/proto"
+	"github.com/dubbogo/gost/log/logger"
```
Comment (FinalT) [inline config-yaml/go-client/cmd/main.go:25]: ditto.

File: start_integrate_test.sh [hunk 4]
```diff
@@ -107,6 +107,9 @@ array+=("compatibility/registry/servicediscovery/nacos")
 array+=("compatibility/registry/all/zookeeper")
 array+=("compatibility/registry/all/nacos")
 
+# config-yaml
+array+=("config-yaml")
+
```
Comment (FinalT) [inline start_integrate_test.sh:112]: Use `config_yaml` instead of `config-yaml`

**Review by FinalT — COMMENTED** [review]
Thanks! A few comments.

**Comment by YarBor** [comment]
ok

**Pull request closed without merging**